src/components/modal/ModalStation/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import ModalWindow from '../ModalWindow';
+import ModalWindowNavigationContainer from '../ModalWindow/ModalWindowNavigationContainer';
+import ModalWindowNavigationListItem from '../ModalWindow/ModalWindowNavigationListItem';
+import Title from '../../common/Title';
+import SelectStationContainer from './SelectStation';
+import { stationData } from '../../../constants/stationData';
+import { IStationData } from '../../../types/models/models';
+
+function ModalStation() {
+  const [data, setData] = useState(stationData);
+  const [currentStep, setCurrentStep] = useState(0);
+
+  const navItems = [
+    'Station Type',
+    'General Information',
+    'Station Amenities',
+    'Connect your Bank',
+    'Connect your System',
+    'Finalize',
+  ];
```

Comment:
Object is more readable and easy to handle
```suggestion
  enum NavItems = {
    STATION_TYPE = 'Station Type',
    ...
  }
```